src/app/main/components/header/header.component.scss

```diff
@@ -0,0 +1,108 @@
+$grey: #807e7b;
```

Comment:
So, $grey... and what? Please, create separated variables.scss and put all variables like this into this file